app/main.js

```diff
@@ -122,10 +148,17 @@ function shutdownDaemon(evenIfNotStartedByApp = false) {
       console.log('Killed lbrynet-daemon process');
     });
   } else if (evenIfNotStartedByApp) {
-    console.log('Killing lbrynet-daemon, even though app did not start it');
-    client.request('daemon_stop', []);
-    // TODO: If the daemon errors or times out when we make this request, find
-    // the process and force quit it.
+    console.log('Stopping lbrynet-daemon, even though app did not start it');
+    client.request('daemon_stop', [], (err, res) => {
+      if (err) {
+        // We could get an error because the daemon is already stopped (good)
```

Comment:
im still confused. we have `client`, which is the connection to the daemon that we use for everything. we call `daemon_stop` on it. your comment says `we could get an error because the daemon is already stopped`. if the daemon is already stopped, then we're good, right? why do we then try to kill all the running daemons?